### PR TITLE
update readme-docs, clean of unnecessary links

### PR DESCRIPTION
**Pull request by krzysztofkozyra021**
## What type of PR is this? (check all applicable)

- [ ] 🚀 Added Name
- [ ] ✨ Feature
- [ ] ✅ Joined Community
- [ ] 🌟 ed the repo
- [ ] 🐛 Grammatical Error
- [x ] 📝 Documentation Update
- [ ] 🚩 Other

## Description

Hello, i removed some of the profiles that didn't add anything to your repo, also i checked for archived, deleted, moved, changed repos of other people, found few with 404 error, also the pages to tutorial which were weird or deleted. 
To sum up, just cleaned of unnecessary links.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,36 +52,32 @@
 ## Categories
 
 #### GitHub Actions 🤖
-- [Abhishek Naidu](https://github.com/abhisheknaiidu/abhisheknaiidu)
+- [Abhishek Naidu](https://github.com/abhisheknaiidu/abhisheknaiidu)
```
Comment (andrzejsydor) [inline README.md:55]: why such a change?

**Review by andrzejsydor — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Cysiuu) [reply README.md:55]: Oh sorry, didn't see it was profile of the repo owner! My bad!

Comment (Cysiuu) [reply README.md:55]: > why such a change?

Oh sorry, didn't see it was profile of the repo owner! My bad!

**Review by Pankeking — APPROVED** [review]
